adapters/mediasquare/mediasquare_test.go

```diff
@@ -0,0 +1,29 @@
+package mediasquare
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/adapters"
+	"github.com/prebid/prebid-server/v3/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBidderMediasquare(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderMediasquare, config.Adapter{
+		Endpoint: "https://pbs-front.mediasquare.fr/msq_prebid"},
+		config.Server{ExternalUrl: "https://pbs-front.mediasquare.fr/msq_prebid", GvlID: 1, DataCenter: "2"})
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	// MakeRequests : case request is empty.
+	resp, errs := bidder.MakeRequests(nil, nil)
+	expectingErrors := []error{errorWritter("<MakeRequests> request", nil, true)}
+	assert.Equal(t, []*adapters.RequestData(nil), resp, "resp, was supposed to be empty result.")
+	assert.Equal(t, expectingErrors, errs, "errs, was supposed to be :", expectingErrors)
+
```

Comment:
This should be tested in JSON files.

Reply:
I initially would have if it was possible,
```
// MakeRequests : case request is empty.
resp, errs := bidder.MakeRequests(nil, nil)
```
Here is a screen-shot of the following test :
![nil-imp](https://github.com/user-attachments/assets/01313a40-e761-41a3-9ef8-bce7e78c48c3)

Cordially

Reply:
I have had, let this test be :
![image](https://github.com/user-attachments/assets/6d0ddeb1-8ec9-4ee6-823e-9f7dacf87a5c)
Tell me if you want me to take it back, it will reduce the coverage by 2% or so... But it is not critical.